lib/src/body.dart

```diff
@@ -5,6 +5,7 @@
 import 'dart:async';
 import 'dart:convert';
 
+import 'package:collection/collection.dart';
```

Comment:
not sure if you care about sorting imports, but mentioning in case you do.
(Personally I don't think it's worth worrying about)